### PR TITLE
tests: use clarinet-sdk to tests the smart contract

### PR DESCRIPTION
**Pull request by hugocaillard**

### CODE REVIEW

File: tests/counter.test.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import { tx } from "@hirosystems/clarinet-sdk";
+import { Cl, UIntCV } from "@stacks/transactions";
+import { describe, expect, it } from "vitest";
```
Comment (moodmosaic) [inline tests/counter.test.ts:3]: It’s nice that the test-discovery mechanism (and the test-runner) are finally decoupled in this new Clarinet version.

/cc @friedger @setzeus

**Review by moodmosaic — COMMENTED** [review]
LGTM!

**Comment by hugocaillard** [comment]
Thanks @moodmosaic!

I'll merge this PR once clarinet-sdk v1 is released (still polishing the docs but we are almost there)

**Comment by moodmosaic** [comment]
🥇 🚀

**Pull request merged**